package.json

```diff
@@ -31,8 +33,10 @@
     "npm-run-all": "^1.2.4",
     "react": "^0.13.2",
     "react-router": "^0.13.3",
-    "webpack": "^1.8.11",
-    "dotenv": "~1.1.0"
+    "webpack": "^1.8.11"
+  },
+  "devDependencies": {
```

Comment:
this is good - quick note for the future a few others can probably be moved under devDependencies (linters, etc)